src/styles/theming/sap_fiori_3_dark.scss

```diff
@@ -268,4 +268,18 @@
   --fdScrollbar_Thumb_Color: var(--sapScrollBar_FaceColor);
   --fdScrollbar_Thumb_Hover_Color: var(--sapScrollBar_Hover_FaceColor);
   --fdScrollbar_Dimension: var(--sapScrollBar_Dimension);
+
+  /* Progress Indicator */
+  --fdProgress_Indicator_Background_Color_Neutral: var(--sapNeutralElementColor);
+  --fdProgress_Indicator_Background_Color_Informative: var(--sapInformativeElementColor);
+  --fdProgress_Indicator_Background_Color_Positive: var(--sapPositiveElementColor);
+  --fdProgress_Indicator_Background_Color_Negative: var(--sapNegativeElementColor);
+  --fdProgress_Indicator_Background_Color_Critical: var(--sapCriticalElementColor);
+  --fdProgress_Indicator_Border_Right: none;
+  --fdProgress_Icon_Offset_Right: 0;
+  --fdProgress_Icon_Offset_Left: 0;
+  --fdProgress_Icon_Informative: '';
```

Comment:
In `sap_fiori_3_hcb.scss` icons are set and here is not, is it intended?

Reply:
Yes. In light and dark themes the icons have colors which correspond to the state, but in HCB and HCW the we don't have state colors and the state is represented by an icon. That's why this icon is not shown in dark and light theme, but only visible in the contrast themes.

Reply:
<img width="1786" alt="Screen Shot 2021-11-15 at 1 45 31 PM" src="https://user-images.githubusercontent.com/39598672/141837158-6225d278-483b-4c40-a41e-59675bc1acfd.png">
<img width="1782" alt="Screen Shot 2021-11-15 at 1 45 51 PM" src="https://user-images.githubusercontent.com/39598672/141837175-1b67ffae-fa97-4e0e-9c19-5261d102ccd7.png">